src/static/css/cellml.css

```diff
@@ -129,14 +133,14 @@ li {
 
 .heading2 {
     color: #444444;
-    font-family: "Roboto Slab","ff-tisa-web-pro","Georgia",Arial,sans-serif;
+    font-family: "Roboto Slab", "ff-tisa-web-pro", "Georgia", "Arial", sans-serif;
```

Comment:
If we are to use `"` for all the font names, then we need to use some too for `sans-serif` (same for `.heading3` below).

Reply:
Umm ... no, I don't think so?  I think that there are some which must be without quotes, whereas for others they're optional?  See the answer to this: https://stackoverflow.com/questions/13751412/why-would-font-names-need-quotes

Reply:
I stand corrected. Learning everyday!